DEVELOPMENT.md

```diff
@@ -16,3 +23,30 @@ You can run the unit tests by simply doing
 ```bash
 make test
 ```
+
+## How to install the controller
+
+You can install the CRDs and the controller by simply doing
+
+```bash
+# Install CRDs into a cluster
+make install
+# Deploy controller in the configured Kubernetes cluster in ~/.kube/config
+make deploy
+```
+
+## How to run the controller locally
+
+You can run the controller on your host by
+
+```bash
+make run
+```
+
+## How to generate and update CRDs API reference documentation
+
+You can update CRDs API reference doc by
+
+```bash
+make gen-crd-api-reference-docs
+```
```

Comment:
Maybe also add a note here somewhere about referring to the Makefile. All the build targets have comments about what they do.